src/client/components/logout/logout.component.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import Popup from 'reactjs-popup';
+import './logout.style.css';
+import PropTypes from 'prop-types';
+
+const userName = 'Servet';
+
+export default function Logout() {
+  const popupView = (closeAction) => (
+    <PopupModal title="LOG OUT" closeAction={closeAction}>
+      <LogoutContent>
+        <LogoutActions closeAction={closeAction} />
+      </LogoutContent>
+    </PopupModal>
+  );
+
+  return (
+    <Popup trigger={PopupTrigger} modal="true">
```

Comment:
modal
(without the true)

Reply:
Means modal=""  or <Popup trigger={PopupTrigger} > ?

Reply:
just modal